### ISSUE TITLE
Biomass Harvest not communicating with succession for partial cohort disturbance

### ISSUE THREAD

**Issue by rmscheller** [LANDIS-II-Foundation/Extension-Biomass-Harvest#13]
A colleague contacted me with this bug.

When a partial cohort removal is performed, the biomass harvested should be handled by the succession extension such that the appropriate amount of biomass is removed from the site or left on site.  For example, typically all leaves are left on site along with 15% of the wood (branches).  This functionality seems to have been lost with the updates to the biomass libraries.

**Comment by rmscheller** [comment]
An update:  After working at it diligently, I got it to work.  However, doing so required large changes to library-biomass-cohorts and library-leaf-biomass-cohorts and some of the harvest libraries.  It is now being tested by a colleague in Japan.   In effect, I had to add methods to record PartialMortality.  Previously, it was only recording complete mortality and then succession acted on it.  

**To do:**  Following disturbance, dead biomass (not just cohorts) is also supposed to be impacted and again, I believe it's only doing so when there is cohort mortality.  This is less critical as harvesting typically has less of an effect on dead material, but it is a flaw.
